### PR TITLE
ETFE-1924: Create CheckYourAnswersConsignee page

### PR DESCRIPTION
**Pull request by joenjones07**

### CODE REVIEW

**Comment by grahampaulcook** [comment]
@joenjones07 Needs rebasing

File: app/viewmodels/sections/checkAnswers/transportArranger/TransportArrangerNameSummary.scala
Comment (james-work-account) [inline app/viewmodels/sections/checkAnswers/transportArranger/TransportArrangerNameSummary.scala:1]: Undo this change.

File: app/viewmodels/sections/checkAnswers/transportArranger/TransportArrangerSummary.scala
Comment (james-work-account) [inline app/viewmodels/sections/checkAnswers/transportArranger/TransportArrangerSummary.scala:1]: Undo this change.

File: app/navigation/ConsigneeNavigator.scala
Comment (james-work-account) [inline app/navigation/ConsigneeNavigator.scala:1]: This was (loosely) organised so that the earlier pages in the flow were at the top. Why was it changed back? It's now much harder to follow what comes next, as the ConsigneeAddressPage (the last page before CYA) is now at the top of the file...

File: app/viewmodels/sections/checkAnswers/transportArranger/TransportArrangerVatSummary.scala
Comment (james-work-account) [inline app/viewmodels/sections/checkAnswers/transportArranger/TransportArrangerVatSummary.scala:1]: Undo this change.

File: app/views/sections/consignee/ConsigneeExciseView.scala.html
Comment (james-work-account) [inline app/views/sections/consignee/ConsigneeExciseView.scala.html:1]: This change reintroduces a WCAG error. Please undo this change.

File: app/views/sections/info/LocalReferenceNumberView.scala.html
Comment (james-work-account) [inline app/views/sections/info/LocalReferenceNumberView.scala.html:1]: Undo this change.

File: conf/messages.cy [hunk 1]
```diff
@@ -324,7 +326,6 @@ transportArrangerVat.checkYourAnswers.label = VAT registration number
 transportArrangerVat.change.hidden = VAT registration number
 
 transportArrangerAddress.checkYourAnswers.label = Address
-transportArrangerAddress.checkYourAnswers.notProvided = {0} section not complete
```
Comment (james-work-account) [inline conf/messages.cy:327]: Undo this change.

File: project/CodeCoverageSettings.scala [hunk 2]
```diff
@@ -16,7 +16,6 @@ object CodeCoverageSettings {
     "testOnly.*",
     "testOnlyDoNotUseInAppConf.*",
     ".*featureswitch.*",
-    ".*pages.*",
```
Comment (james-work-account) [inline project/CodeCoverageSettings.scala:19]: Undo this change.

File: test-utils/fixtures/messages/BaseMessages.scala [hunk 3]
```diff
@@ -39,7 +39,6 @@ trait BaseMessages { _: i18n =>
   val no: String = "No"
   val change: String = "Change"
   val remove: String = "Remove"
-  val sectionNotComplete: String => String = section => s"$section section not complete"
```
Comment (james-work-account) [inline test-utils/fixtures/messages/BaseMessages.scala:42]: Undo this change.

**Review by james-work-account — APPROVED** [review]

**Review by grahampaulcook — APPROVED** [review]
